nifi-extension-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/ml/textract/GetAwsTextractJobStatus.java

```diff
@@ -111,7 +112,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             } else {
                 throw new IllegalStateException("Unrecognized job status");
             }
-        } catch (final ThrottlingException e) {
+        } catch (final ThrottlingException | ProvisionedThroughputExceededException e) {
             getLogger().info("Request Rate Limit exceeded", e);
             session.transfer(flowFile, REL_THROTTLED);
```

Comment:
Thank for adding the additional exception @tpalfy! It makes sense.
I think `yield()` should also be called in this case in order to avoid subsequent requests when the rate limit is already exceeded.